src/snap.c

```diff
@@ -0,0 +1,284 @@
+// SPDX-License-Identifier: GPL-2.0-only
+#include <assert.h>
+#include <strings.h>
+#include "labwc.h"
+#include "workspaces.h"
+#include "view.h"
+#include "snap.h"
```

Comment:
Nitpick: alphabetic ordering for our includes would be nice.

Reply:
Done (your [fixup])